source/host/KBindings.cc

```diff
@@ -0,0 +1,245 @@
+#include "Common.hh"
+
+#include "game/system/KPadController.hh"
+
+#include "game/kart/KartObjectProxy.hh"
+
+#include <host/KBindSystem.hh>
```

Comment:
These have to be changed since you moved which folder this is present in
```suggestion
#include "host/KBindSystem.hh"

#include <game/kart/KartObjectProxy.hh>
```